constants/map_constants.asm

```diff
@@ -1,9 +1,9 @@
-newgroup: macro
+newgroup: MACRO
 const_value = const_value + 1
 	enum_start 1
 endm
```

Comment:
`endm` should match `macro` case

Reply:
my bad, forgot those.

Reply:
wen't ahead and removed random indents too #50fc9c3